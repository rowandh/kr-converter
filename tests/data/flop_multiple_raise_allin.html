
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-3-29536332</p></td>
                                <td height="26" ><p align="center">&nbsp;2025-02-05 오후 3:10:16</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>Kojhgbvfffghgvg (K 원 페어)</b></p></td>
<!--								<td ><p align="center">&nbsp;1351413</p></td>   -->
								<td ><p align="center">&nbsp;1,351,413</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>fdgksuik</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>K <font color='red'>◆</font>Q <font color='red'>♥</font>J ♣2 <font color='red'>◆</font>9 <font color='red'>◆</font>A ♣3 [A 탑]  (Big Blind)
<br>* 시작 : [StageNo:29536332] [Credit:651,023원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:150]
<br>* NICKNAME:[fdgksuik]
<br>* 앤티: -5,000원(646,023원)
<br>* 홀 카드딜: <font color='red'>♥</font>K(38) <font color='red'>◆</font>Q(24) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(<font color='red'>♥</font>K <font color='red'>◆</font>Q)]
<br>* 베팅: [블라인드:BIG] [금액:5,000원] [Creadit:641,023원]
<br>* 베팅: [풀] (355,000원) Credit(286,023원) - 베팅순서: [0][9]
[5384ms]
<br>* 베팅: [올인] (286,023원) Credit(0원) - 베팅순서: [0][14]
[1329ms]
<br>* 쇼다운 상태 진입: [턴:0]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>K<font color='red'>◆</font>Q) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[0원]
<br>* 결과: 패배 [족보:A 탑] [카드:<font color='red'>◆</font>A <font color='red'>♥</font>K <font color='red'>◆</font>Q <font color='red'>♥</font>J <font color='red'>◆</font>9]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-651,023</font></p></td>
											<td style="word-break:break-all"><p align="right">0</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>mdsfdfoghs</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣7 <font color='red'>◆</font>4 [7 탑]  (다이) (Small Blind)
<br>* 시작 : [StageNo:29536332] [Credit:750,000원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:150]
<br>* NICKNAME:[mdsfdfoghs]
<br>* 앤티: -5,000원(745,000원)
<br>* 홀 카드딜: ♣7(45) <font color='red'>◆</font>4(16) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(♣7 <font color='red'>◆</font>4)]
<br>* 베팅: [블라인드:SMALL] [금액:5,000원] [Creadit:740,000원]
<br>* 베팅: 다이 [0](740,000원) - 베팅순서: [0][8]
[499ms]
<br>* 커뮤니티 카드 딜: H(♣7<font color='red'>◆</font>4) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[740,000원]
<br>* 결과: 패배 [족보:7 탑] [카드:♣7 <font color='red'>◆</font>4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">740,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>85xcdkfoqs</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣9 ♣8 [9 탑]  (다이) 
<br>* 시작 : [StageNo:29536332] [Credit:1,535,760원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[85xcdkfoqs]
<br>* 앤티: -5,000원(1,530,760원)
<br>* 홀 카드딜: ♣9(47) ♣8(46) [9 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:9 탑(♣9 ♣8)]
<br>* 베팅: 콜 -5,000원(1,525,760원) - 베팅순서: [0][1]
[2421ms]
<br>* 베팅: 다이 [0](1,525,760원) - 베팅순서: [0][10]
[2601ms]
<br>* 커뮤니티 카드 딜: H(♣9♣8) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[1,525,760원]
<br>* 결과: 패배 [족보:9 탑] [카드:♣9 ♣8] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">1,525,760</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Piurdkgdx</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠J ♠10 [J 탑]  (다이) 
<br>* 시작 : [StageNo:29536332] [Credit:931,490원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:200]
<br>* NICKNAME:[piurdkgdx]
<br>* 앤티: -5,000원(926,490원)
<br>* 홀 카드딜: ♠J(10) ♠10(9) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(♠J ♠10)]
<br>* 베팅: 콜 -5,000원(921,490원) - 베팅순서: [0][2]
[1092ms]
<br>* 베팅: 다이 [0](921,490원) - 베팅순서: [0][11]
[1519ms]
<br>* 커뮤니티 카드 딜: H(♠J♠10) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[921,490원]
<br>* 결과: 패배 [족보:J 탑] [카드:♠J ♠10] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">921,490</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>eskhvmop</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣J <font color='red'>◆</font>10 [J 탑]  (다이) 
<br>* 시작 : [StageNo:29536332] [Credit:985,506원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[kgtrdsl]
<br>* 앤티: -5,000원(980,506원)
<br>* 홀 카드딜: ♣J(49) <font color='red'>◆</font>10(22) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(♣J <font color='red'>◆</font>10)]
<br>* 베팅: 다이 [0](980,506원) - 베팅순서: [0][5]
[1443ms]
<br>* 커뮤니티 카드 딜: H(♣J<font color='red'>◆</font>10) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[980,506원]
<br>* 결과: 패배 [족보:J 탑] [카드:♣J <font color='red'>◆</font>10] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">980,506</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>dygfdasd</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>6 ♠3 [6 탑]  (다이) 
<br>* 시작 : [StageNo:29536332] [Credit:506,820원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[dygfdasd]
<br>* 앤티: -5,000원(501,820원)
<br>* 홀 카드딜: <font color='red'>♥</font>6(31) ♠3(2) [6 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:6 탑(<font color='red'>♥</font>6 ♠3)]
<br>* 베팅: 다이 [0](501,820원) - 베팅순서: [0][7]
[203ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>6♠3) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[501,820원]
<br>* 결과: 패배 [족보:6 탑] [카드:<font color='red'>♥</font>6 ♠3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">501,820</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>jdbdbhkf</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠9 <font color='red'>♥</font>9 [9 원 페어]  (다이) 
<br>* 시작 : [StageNo:29536332] [Credit:430,576원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:200]
<br>* NICKNAME:[jdbdbhkf]
<br>* 앤티: -5,000원(425,576원)
<br>* 홀 카드딜: ♠9(8) <font color='red'>♥</font>9(34) [9 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:9 원 페어(♠9 <font color='red'>♥</font>9)]
<br>* 베팅: 콜 -75,000원(350,576원) - 베팅순서: [0][6]
[1178ms]
<br>* 베팅: 다이 [0](350,576원) - 베팅순서: [0][13]
[6338ms]
<br>* 커뮤니티 카드 딜: H(♠9<font color='red'>♥</font>9) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[350,576원]
<br>* 결과: 패배 [족보:9 원 페어] [카드:♠9 <font color='red'>♥</font>9] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-80,000</font></p></td>
											<td style="word-break:break-all"><p align="right">350,576</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Kojhgbvfffghgvg</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>K ♣K <font color='red'>♥</font>J ♣2 <font color='red'>◆</font>9 <font color='red'>◆</font>A ♣3 [K 원 페어]  
<br>* 시작 : [StageNo:29536332] [Credit:736,549원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[kojhgbvfffghgvg]
<br>* 앤티: -5,000원(731,549원)
<br>* 홀 카드딜: <font color='red'>◆</font>K(25) ♣K(51) [K 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:K 원 페어(<font color='red'>◆</font>K ♣K)]
<br>* 베팅: [풀] (75,000원) Credit(656,549원) - 베팅순서: [0][4]
[2396ms]
<br>* 베팅: [올인] (656,549원) Credit(0원) - 베팅순서: [0][12]
[726ms]
<br>* 쇼다운 상태 진입: [턴:0]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>K♣K) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
# 공베팅 반환 [85,526원]<br>* 종료: WinMoney[1,351,413원] Credit[1,436,939원]
<br>* 결과: 승리 [족보:K 원 페어] [카드:<font color='red'>◆</font>K ♣K <font color='red'>◆</font>A <font color='red'>♥</font>J <font color='red'>◆</font>9]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>700,390</font></p></td>
											<td style="word-break:break-all"><p align="right">1,436,939</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>quaibuderjkd</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>5 ♠4 [5 탑]  (다이) 
<br>* 시작 : [StageNo:29536332] [Credit:806,290원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:150]
<br>* NICKNAME:[quaibuderjkd]
<br>* 앤티: -5,000원(801,290원)
<br>* 홀 카드딜: <font color='red'>◆</font>5(17) ♠4(3) [5 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:5 탑(<font color='red'>◆</font>5 ♠4)]
<br>* 베팅: 다이 [0](801,290원) - 베팅순서: [0][3]
[210ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>5♠4) C (<font color='red'>♥</font>J♣2<font color='red'>◆</font>9) (<font color='red'>◆</font>A) (♣3)
<br>* 종료: WinMoney[0원] Credit[801,290원]
<br>* 결과: 패배 [족보:5 탑] [카드:<font color='red'>◆</font>5 ♠4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">801,290</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-3-29536332">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


