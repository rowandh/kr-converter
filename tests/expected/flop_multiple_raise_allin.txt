PokerStars Hand #15329536332:  Hold'em No Limit ($5000/$5000) - 2025/02/05 15:10:16 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: 85xcdkfoqs ($1535760 in chips)
Seat 2: Piurdkgdx ($931490 in chips)
Seat 3: quaibuderjkd ($806290 in chips)
Seat 4: Kojhgbvfffghgvg ($736549 in chips)
Seat 5: eskhvmop ($985506 in chips)
Seat 6: jdbdbhkf ($430576 in chips)
Seat 7: dygfdasd ($506820 in chips)
Seat 8: mdsfdfoghs ($750000 in chips)
Seat 9: fdgksuik ($651023 in chips)
85xcdkfoqs: posts the ante $5000
Piurdkgdx: posts the ante $5000
quaibuderjkd: posts the ante $5000
Kojhgbvfffghgvg: posts the ante $5000
eskhvmop: posts the ante $5000
jdbdbhkf: posts the ante $5000
dygfdasd: posts the ante $5000
mdsfdfoghs: posts the ante $5000
fdgksuik: posts the ante $5000
mdsfdfoghs: posts small blind $5000
fdgksuik: posts big blind $5000
*** HOLE CARDS ***
85xcdkfoqs: calls $5000
Piurdkgdx: calls $5000
quaibuderjkd: folds
Kojhgbvfffghgvg: raises $70000 to $75000
eskhvmop: folds
jdbdbhkf: calls $75000
dygfdasd: folds
mdsfdfoghs: folds
fdgksuik: raises $285000 to $360000
85xcdkfoqs: folds
Piurdkgdx: folds
Kojhgbvfffghgvg: raises $371549 to $731549 and is all-in
jdbdbhkf: folds
fdgksuik: calls $286023 and is all-in
Uncalled bet ($85526) returned to Kojhgbvfffghgvg
*** FLOP *** [Jh 2c 9d]
*** TURN *** [Jh 2c 9d] [Ad]
*** RIVER *** [Jh 2c 9d] [Ad] [3c]
Kojhgbvfffghgvg collected $1351413 from pot
*** SUMMARY ***
Total pot $1427046 | Rake $75633
Board [Jh 2c 9d Ad 3c]
Seat 1: 85xcdkfoqs mucked [9c 8c]
Seat 2: Piurdkgdx mucked [Js Ts]
Seat 3: quaibuderjkd mucked [5d 4s]
Seat 4: Kojhgbvfffghgvg showed [Kd Kc] and won ($1351413)
Seat 5: eskhvmop mucked [Jc Td]
Seat 6: jdbdbhkf mucked [9s 9h]
Seat 7: dygfdasd mucked [6h 3s]
Seat 8: mdsfdfoghs mucked [7c 4d]
Seat 9: fdgksuik showed [Kh Qd] and lost